ocs_ci/ocs/fiojob.py

```diff
@@ -376,15 +376,25 @@ def workload_fio_storageutilization(
     # BZ 1775432 and check that there is no more recent BZ or JIRA in this
     # area)
     ceph_full_ratios = [
-        'mon_osd_full_ratio',
-        'mon_osd_backfillfull_ratio',
-        'mon_osd_nearfull_ratio',
+        'full_ratio',
+        'backfillfull_ratio',
+        'nearfull_ratio',
```

Comment:
please save in constants.py

Reply:
I don't think this make any sense.

Reply:
aren't these applicable for any time we need to fetch the ceph_ratios from `ceph osd dump` output?

Reply:
Yes, but it would only increased complexity without any benefit.

That said, I could move this piece of code into separate function during work on #1861.

Reply:
ok, thanks. In any case, it's not a big deal